core/web/JsonLd.php

```diff
@@ -80,14 +133,26 @@ public static function person(array $config = [])
     /**
      * Register new Place
      *
-     * @param array $config Optional config array to provided person data via setter methods.
+     * @param array $config Optional config array to provided place data via setter methods.
      *
      * @return \luya\web\jsonld\Place
      */
     public static function place(array $config = [])
     {
         return self::addGraph((new Place($config)));
     }
+
+    /**
+     * Register new Social Media Posting.
+     *
+     * @param array $config Optional config array to provided social media posting data via setter methods.
```

Comment:
Missing @since 1.0.1 tag

Reply:
I'll walk through and add it in the next few days